endpoints/openrtb2/auction.go

```diff
@@ -442,6 +442,7 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request, labels *metric
 		errs = []error{err}
 		return
 	}
+	labels.RequestSize = len(requestJson)
```

Comment:
It looks like we are capturing the size of the request that comes in over the wire prior to merging in stored requests for both the auction and video endpoints while the amp endpoint captures the request size after stored requests are merged. IMO this is a bit confusing. Do we even need the amp request size since it is just based on stored requests?

Reply:
Modified to not log these metrics oh the `amp` endpoint